src/sdk/object_sdk.js

```diff
@@ -689,6 +698,8 @@ class ObjectSDK {
                 const ns = await this._get_bucket_namespace(params.bucket);
                 this._check_is_readonly_namespace(ns);
                 if (params.copy_source) await this.fix_copy_source_params(params, ns);
+                params.force_md5_etag_bucket = await this.read_force_md5_etag_for_bucket(params.bucket);
+                params.force_md5_etag_account = await this.read_force_md5_etag_for_account();
```

Comment:
I'm not sure there is a reason not to send a single force_md5 flag, WDYT?
```suggestion
                params.force_md5_etag = await this.read_force_md5_etag_for_bucket(params.bucket) || await this.read_force_md5_etag_for_account();
```

Reply:
Yes, that would be better I think. We can remove some unecessary checks in that way.